config/initializers/diaspora_federation.rb

```diff
@@ -0,0 +1,47 @@
+# configure the federation engine
+DiasporaFederation.configure do |config|
+  # the pod url
+  config.server_uri = AppConfig.pod_uri
+
+  config.define_callbacks do
+    on :person_webfinger_fetch do |handle|
+      person = Person.find_local_by_diaspora_handle(handle)
+      if person
+        DiasporaFederation::WebFinger::WebFinger.new(
+          acct_uri:    "acct:#{person.diaspora_handle}",
+          alias_url:   url_to("/people/#{person.guid}"),
```

Comment:
I would consider moving this into a model method, like we already have `profile_url` and such.